site/content/en/docs/tasks/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -0,0 +1,611 @@
+      "positionInLocalQueue": 4
+    },
+    {
+      "metadata": {
+        "name": "job-sample-job-44nts-c01a4",
+        "namespace": "default",
+        "creationTimestamp": "2023-12-05T15:42:04Z",
+        "ownerReferences": [
+          {
+            "apiVersion": "batch/v1",
+            "kind": "Job",
+            "name": "sample-job-44nts",
+            "uid": "07275e7b-f5f6-4783-8939-3ba891799222"
+          }
+        ]
+      },
+      "priority": 0,
+      "localQueueName": "user-queue",
+      "positionInClusterQueue": 5,
+      "positionInLocalQueue": 5
+    },
+    {
+      "metadata": {
+        "name": "job-sample-job-f8h9t-a564c",
+        "namespace": "default",
+        "creationTimestamp": "2023-12-05T15:42:04Z",
+        "ownerReferences": [
+          {
+            "apiVersion": "batch/v1",
+            "kind": "Job",
+            "name": "sample-job-f8h9t",
+            "uid": "0916dca3-309e-4381-9791-b9a450d7f344"
+          }
+        ]
+      },
+      "priority": 0,
+      "localQueueName": "user-queue",
+      "positionInClusterQueue": 6,
+      "positionInLocalQueue": 6
+    }
+  ]
+}
+```
+
+You can pass optional query parameters:
+- limit `<integer>` - 1000 on default. It indicates max number of pending workloads that should be fetched.
+- offset `<integer>` - 0 by default. It indicates position of the first pending workload that should be fetched, starting from 0.
+
+To view only the first 2 pending workloads use:
+```shell
+kubectl get --raw "/apis/visibility.kueue.x-k8s.io/v1alpha1/clusterqueues/cluster-queue/pendingworkloads?limit=3"
+```
+You should get results similar to 
+``` json
+{
+  "kind": "PendingWorkloadsSummary",
+  "apiVersion": "visibility.kueue.x-k8s.io/v1alpha1",
+  "metadata": {
+    "creationTimestamp": null
+  },
+  "items": [
+    {
+      "metadata": {
+        "name": "job-sample-job-5kb24-1a02c",
+        "namespace": "default",
+        "creationTimestamp": "2023-12-05T15:42:03Z",
+        "ownerReferences": [
+          {
+            "apiVersion": "batch/v1",
+            "kind": "Job",
+            "name": "sample-job-5kb24",
+            "uid": "f983df07-fdab-4e8b-81a6-a537d8225062"
+          }
+        ]
+      },
+      "priority": 0,
+      "localQueueName": "user-queue",
+      "positionInClusterQueue": 0,
+      "positionInLocalQueue": 0
+    },
+    {
+      "metadata": {
+        "name": "job-sample-job-jrjfr-8d56e",
+        "namespace": "default",
+        "creationTimestamp": "2023-12-05T15:42:03Z",
+        "ownerReferences": [
+          {
+            "apiVersion": "batch/v1",
+            "kind": "Job",
+            "name": "sample-job-jrjfr",
+            "uid": "5863cf0e-b0e7-43bf-a445-f41fa1abedfa"
+          }
+        ]
+      },
+      "priority": 0,
+      "localQueueName": "user-queue",
+      "positionInClusterQueue": 1,
+      "positionInLocalQueue": 1
+    }
+  ]
+}
+```
+
+To view pending workloads having at least position 5 in Cluster Queue use:
```

Comment:
Too many examples that don't add much value. Mix this within the previous example.